test/unit-tests.ts

```diff
@@ -0,0 +1,23 @@
+import * as assert from "assert";
+import * as functions from "./../src/functions/functions";
+import * as mocha from "mocha";
+
+describe("Test Excel Custom Functions", function () {
+    describe("Test add function", function () {
+        it(`Add function should return expected value`, async function () {
```

Comment:
Consider: "add two numbers"  as the test case name. Other tests cases would be "add two negative numbers", "add zero to a number", etc. This is a better pattern for naming test cases. Granted there is only one simple test case here, but the idea in practice is these get built out. If you wanted to add a few different test cases to illustate this, it would be helpful.

Also, one more realistic case that affects Excel is a number that is too large. If you add 1 to number.MAX_VALUE (not sure where this constant actually lives), I think this results in positive infinity which is `number.NaN`.